blackbox/docs/appendices/release-notes/2.3.0.txt

```diff
@@ -0,0 +1,169 @@
+
+.. _rolling upgrade: http://crate.io/docs/crate/guide/best_practices/rolling_upgrade.html
+.. _full restart upgrade: http://crate.io/docs/crate/guide/best_practices/full_restart_upgrade.html
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Changelog
+=========
+
+Breaking Changes
+----------------
+
+- Certain metrics in the ``sys.nodes`` table have been deprecated and now
+  always return ``-1``, except network metrics with still return ``0`` as
+  previously when Sigar was not available.
+
+  The affected metrics are: network metrics, read/write filesystem metrics for
+  individual disks, system/user/idle/stolen values for CPU, and system/user
+  values for process CPU.
+
+  This is due to the removal of the "Sigar" library dependency.
+
+- The default value of the setting ``auth.host_based.enabled`` (``false``) is
+  overwritten with ``true`` in the ``crate.yml`` that is shipped with the
+  tarball and Linux distributions of CrateDB and also contains a sane default
+  configuration for ``auth.host_based.config``.
+
+  This is done in order to provide a better default installation for new users
+  without breaking exising HBA configurations.
+
+  An implication of these settings is that whenever Enterprise is enabled
+  (default behaviour) connections from remote hosts *always* require
+  password authentication. Note, that when running CrateDB in Docker, the host
+  of the Docker container is also considered a remote host.
+
+- Columns aren't implicitly cast to a type anymore. Whenever columns are
+  compared to Literals (e.g. 'string', 1, 1.2), these literals will be
+  converted to the column type but not vice-versa. The column can still be
+  manually cast to a type by using a cast function.
+
+- Table ``information_schema.table_constraints`` is now returning
+  ``constraint_name`` as type string instead of type array. Constraint type
+  ``PRIMARY_KEY`` has been changed to ``PRIMARY KEY``. Also PRIMARY KEY
+  constraint is not returned when not explicitly defined.
+
+Changes
+-------
+
+- Update the Admin UI to use new CPU metrics for its graphs.
+
+- Added the "password" authentication method which is available for connections
+  via the PostgreSQL wire protocol and HTTP. This method allows clients to
+  authenticate using a valid database user and its password. For HTTP, the
+  ``X-User`` header, used to provide a username, has been deprecated in favour
+  of the standard HTTP ``Authorization`` header with the Basic Authentication
+  Scheme.
+
+- Added the ``hyperloglog_distinct`` aggregation function. This feature is only
+  available in the Enterprise Edition.
+
+- Added a ``os['cpu']['used']`` column to the ``sys.nodes`` table. This
+  replaces the deprecated system/user/idle/stolen values.
+
+- Subqueries which filter on primary key columns now have the same realtime
+  semantics as the equivalent top-level queries.
+
+- Added support for changing the password of existing users using ``ALTER
```

Comment:
Would it make sense to group/merge this with the related `CREATE USER` entry?

Reply:
yes it makes sense. there are also a few other things that could be ground